interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -1099,8 +1099,8 @@ public function getAllergies($pid)
                 <startdate>" . xmlEscape($row['begdate'] ? preg_replace('/-/', '', $row['begdate']) : "00000000") . "</startdate>
                 <enddate>" . xmlEscape($row['enddate'] ? preg_replace('/-/', '', $row['enddate']) : "00000000") . "</enddate>
                 <reaction_text>" . xmlEscape($reaction_text ? Listener::z_xlt($reaction_text) : "") . "</reaction_text>
-                <reaction_code>" . xmlEscape($reaction_code[1] ?: '') . "</reaction_code>
```

Comment:
I'm curious, did parser complain about the short hand if reation_text was undefined or null? The important thing is that null isn't inserted into string for render. What you have is fine but was curious about warning being issued.

Reply:
it's that php8 warning 
![Screenshot from 2024-01-01 13-34-02](https://github.com/openemr/openemr/assets/5618372/894ce1b7-b987-4995-90a0-bd82cf8201ff)

happy new year @sjpadgett :partying_face:

Reply:
Thanks, same to you!